### PR TITLE
enhance styles

### PR DESCRIPTION
**Pull request by DashrathKY**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -3,9 +3,10 @@
   "name": "PerceptPixel Image Transformer",
   "version": "1.0",
   "permissions": [
-    "storage",
-    "https://api.perceptpixel.com/"
-
+    "storage"
+  ],
+  "host_permissions": [
```
Comment (kaushikm21) [inline manifest.json]: Why is this required?

File: popup.js [hunk 2]
```diff
@@ -179,7 +179,7 @@ document.getElementById('uploadButton').addEventListener('click', () => {
         progressBar.style.width = '0%'; // Reset progress bar width
 
         const xhr = new XMLHttpRequest();
-        xhr.open('POST', 'https://api.perceptpixel.com/media/upload', true);
+        xhr.open('POST', 'http://localhost:8000/api/v1/media/upload', true);
```
Comment (kaushikm21) [inline popup.js]: localhost?

File: manifest.json (hunk 1 above)
Comment (DashrathKY) [reply manifest.json]: I had in the intermediate commits, have removed it before raising the PR. Please check latest changes.

File: popup.js (hunk 2 above)
Comment (DashrathKY) [reply popup.js]: I had it for testing, had removed before raising the PR. Please have a look at the latest changes.

Comment (DashrathKY) [reply popup.js]: You're possibly looking at old/intermediate commits, Please have a look at the final file changes.

**Comment by kaushikm21** [comment]
lgtm
